samigo/samigo-qti/src/java/org/sakaiproject/tool/assessment/qti/helper/AuthoringHelper.java

```diff
@@ -908,6 +912,17 @@ public XmlStringBuffer readXMLDocument(InputStream inputStream)
     try
     {
       DocumentBuilder documentBuilder = builderFactory.newDocumentBuilder();
+
+      documentBuilder.setEntityResolver(new EntityResolver() {
+          @Override
+          public InputSource resolveEntity(String publicId, String systemId) throws SAXException, IOException {
+              if (systemId != null && systemId.endsWith("/ims_qtiasiv1p2p1.dtd")) {
```

Comment:
i'm surprised this custom Override is still needed with the above DTD inclusion

Reply:
Without this method it doesn't work...

![image](https://user-images.githubusercontent.com/15141743/115201973-2fd7c180-a0f6-11eb-8dfa-2f9625fd14b2.png)